.github/workflows/stress.yml

```diff
@@ -17,7 +17,7 @@ env:
 
 jobs:
   stress-test:
-    runs-on: [self-hosted, Linux, X64]
+    runs-on: [ubuntu-latest-4-cores, linux]
```

Comment:
so we're going to have 2 org runner groups with these names that this repo is given access to? why the second?

https://docs.github.com/en/actions/using-github-hosted-runners/using-larger-runners#overview-of-larger-runners

Reply:
Not sure I understand the question.
This is just requiring 2 labels, the `linux` label might be redundant.